README.md

```diff
@@ -1189,10 +1186,31 @@ negotiation.
 
 ## Offers and Negotiations
 
-If you don’t yet have an offer, it’s important to negotiate firmly and fairly to get a
-good one. A Guide like this can’t give you personal advice on what a reasonable offer is,
-as that depends greatly on your skills, the marketplace of candidates, what other offers
-you have, what the company can pay, what other candidates the company has found, and the
+Before accepting an offer, it’s important to negotiate firmly and fairly.
+You’re planning to devote a lot of your time and life to any full-time role, and it’s
+worth making sure it’s a good investment of that time.
+
+☝️ A few people are inexperienced, uncertain, or so anxious about negotiations they avoid
+them as much as they can, sometimes just accepting an offer with little discussion.
+It can indeed be stressful to discuss offers.
+However, even if you feel uncomfortable negotiating an offer with a potential employer, it
+may help to remember why it’s important to have these conversations:
+
+- Firstly, negotiations are important to let you focus on what you actually want.
+  Not being clear with yourself on what things are important to you, be it personal growth,
+  career growth, impact, recognition, cash, ownership, or teamwork.
+- Rushing through this can actually shortchange not just yourself, but others you will work
```

Comment:
Adjust the passive--unnecessarily complex phrasing ("Rushing through this..."). Just tightening these three sentences solves. "If you aren't satisfied with the terms of your offer, accepting it can be tough not just for you but for your company and colleagues as well. No one wants to take on a hire who's going to walk away when something better comes along. For everyone's sake, take your time now to consider what you really want--and then ask for it."

Reply:
👍 Fixed.